lib/avromatic/model/types/abstract_type.rb

```diff
@@ -31,6 +31,10 @@ def coerced?(_value)
         def serialize(_value, **)
           raise "#{__method__} must be overridden by #{self.class.name}"
         end
+
+        def referenced_models
```

Comment:
Should we call this `referenced_model_classes` to make it more obvious that we're not returning instantiated models?